src/osd/osd_types.h

```diff
@@ -1147,6 +1147,9 @@ struct pg_pool_t {
     FLAG_WRITE_FADVISE_DONTNEED = 1<<7, // write mode with LIBRADOS_OP_FLAG_FADVISE_DONTNEED
     FLAG_NOSCRUB = 1<<8, // block periodic scrub
     FLAG_NODEEP_SCRUB = 1<<9, // block periodic deep-scrub
+    FLAG_NO_QUOTA = 1<<10, // pool is currently running out of quota, will set FLAG_FULL too
```

Comment:
maybe this should be called FLAG_FULL_NO_QUOTA since it is only set when FULL is also set?